src/app/containers/MostRead/index.jsx

```diff
@@ -46,14 +46,14 @@ const MostReadContainer = ({
 MostReadContainer.propTypes = {
   mostReadEndpointOverride: string,
   constrainMaxWidth: bool,
-  maxTwoColumns: bool,
+  columnLayout: oneOf(['oneColumn', 'twoColumn', 'multiColumn']),
   isOnFrontPage: bool,
 };
 
 MostReadContainer.defaultProps = {
   mostReadEndpointOverride: null,
   constrainMaxWidth: false,
-  maxTwoColumns: false,
+  columnLayout: 'multiColumn',
```

Comment:
This is already defined here https://github.com/bbc/simorgh/pull/5739/files#diff-04298a8f3ec1cbafcb68a13991abd880R201, I think we can do away with this default prop definition and retain the implementation in `canonical/index.js`